internal/controller/assets/daemonset.go

```diff
@@ -65,6 +67,35 @@ func dsUpdateStrategy(node *falconv1alpha1.FalconNodeSensor) appsv1.DaemonSetUpd
 	return appsv1.DaemonSetUpdateStrategy{Type: appsv1.OnDeleteDaemonSetStrategyType}
 }
 
+func dsAutoPilotDeployAllowlistLabel(node *falconv1alpha1.FalconNodeSensor) map[string]string {
+	if *node.Spec.Node.GKE.Enabled && node.Spec.Node.GKE.DeployAllowListVersion != nil {
+		return map[string]string{
+			common.GKEAutoPilotAllowListLabelKey: fmt.Sprintf("%s-%s", common.GKEAutoPilotDeployDSAllowlistPrefix, *node.Spec.Node.GKE.DeployAllowListVersion),
```

Comment:
question: Do you need the complete path to the yaml file as the value here, similar to how it is being set in Falcon Helm?

Reply:
No need for the full file path.  The filename only is fine.